Adafruit_BMP280.cpp

```diff
@@ -37,8 +37,10 @@ Adafruit_BMP280::~Adafruit_BMP280(void) {
     delete spi_dev;
   if (i2c_dev)
     delete i2c_dev;
-  delete temp_sensor;
-  delete pressure_sensor;
+  if (temp_sensor == nullptr)
+    delete temp_sensor;
+  if (pressure_sensor == nullptr)
+    delete pressure_sensor;
```

Comment:
Can use same syntax as the `spi_dev` and `i2c_dev` checks being done here as well?

Reply:
Added validations and also I did run the local ci-arduino.